js/apps/admin-ui/src/components/dropdown-panel/dropdown-panel.css

```diff
@@ -36,10 +36,8 @@
   z-index: 1;
 }
 
-.light-mode {
-  background-color: var(--pf-v5-global--Color--light-100);
```

Comment:
I think Agi added this code together with some other styles under #30510 to make the custom `.kc-dropdown-panel` look a bit nicer, we should probably add the light background color to the default styles for that reason.

Reply:
That is already there see `kc-dropdown-panel-content` class

Reply:
Totally missed that, consider this resolved.